README.md

```diff
@@ -1 +1,35 @@
-# Capstone1
\ No newline at end of file
+# Youth Week
+
+> a website about the International youth Week event. for the microverse capstone1.
+
+## Built With
+
+- HTML
+- SCSS
+- JavaScript
+
+## Live Project
+
+[Live Link](http://127.0.0.1:5500/index.html)
```

Comment:
- [ ] [REQUIRED] Please make use of GitHub Pages to deploy the web pages and test to see that the link is working. Your current link can only work on your local machine, but making use of Github pages will enable anyone across the network to view your work without cloning the repo to their local machines.😊